nova/live_test.go

```diff
@@ -373,11 +397,17 @@ func (s *LiveTests) TestGetServer(c *gc.C) {
 	s.assertServerDetails(c, server)
 }
 
-func (s *LiveTests) waitTestServerToStart(c *gc.C) {
+func (s *LiveTests) waitTestServerToStart(c *gc.C, id ...string) {
 	// Wait until the test server is actually running
-	c.Logf("waiting the test server %s to start...", s.testServer.Id)
+	var serverId string
```

Comment:
I don't quite get why we are using a slice here for id
It would be better just to take an explicit id arg and wait on that.
Callers would always pass in an id - either a bespoke id or s.testServer.Id

The above is much simpler to implement and understand, even though it means changing a few callers to pass in s.testServer.Id